plugins/com.google.cloud.tools.eclipse.dataflow.ui.test/src/com/google/cloud/tools/eclipse/dataflow/ui/MessagesTest.java

```diff
@@ -14,14 +14,19 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.eclipse.dataflow.ui.page;
+package com.google.cloud.tools.eclipse.dataflow.ui;
 
 import org.junit.Assert;
 import org.junit.Test;
 
 public class MessagesTest {
 
   @Test
+  public void testNewCloudDataflowProject() {
+    Assert.assertEquals("New Cloud Dataflow Project", Messages.getString("new.cloud.dataflow.project"));
```

Comment:
I think this would fail if it was run on a non-english locale where there were translations.  A hypothetical problem at this point!